### PR TITLE
Add annotations to special pages

### PR DESCRIPTION
**Pull request by pwirth**
https://github.com/inblockio/DataAccounting/issues/150

### CODE REVIEW

File: includes/SpecialVerifiedExport.php [hunk 1]
```diff
@@ -30,6 +32,9 @@ public function __construct( ILoadBalancer $loadBalancer = null ) {
 		$this->loadBalancer = $loadBalancer ?? MediaWikiServices::getInstance()->getDBLoadBalancer();
 	}
 
+	/**
+	 * @param string|null $par
```
Comment (rht) [inline includes/SpecialVerifiedExport.php:36]: Is this in the form of doc block because PHP 7.4 doesn't have union operator yet?

Comment (pwirth) [reply includes/SpecialVerifiedExport.php:36]: exactly. php also does not have specified arrays like Title[] or string[] etc.
In this cases we should use a docblock

**Comment by rht** [comment]
Merged, thank you!

**Comment by rht** [comment]
Would be great if the improvements here can be applied to upstream MW too.

**Pull request merged**